src/main/resources/static/api.yml

```diff
@@ -3403,7 +3403,7 @@ components:
         - manager
         - workflow_manager
         - applicant
-        - admin
+        - role_admin
```

Comment:
How come this has been given a prefix of `role_` but the others haven't?

Reply:
The facility granted to holders of this role is managing the roles of other users. So the role is "administrator of roles" or "role admin". As per https://trello.com/c/0CGdMDiu/524-draft-roles-for-the-ap-service#comment-633d9bca5897d900de2ca81b

Clearly this naming is confusing. Though it does seem to be have been adopted and understood as  intended by the CRU staff. It's used on the `Master list - N54 and N55 User List Revised 230123` list used for the seeding.

Can you think of a better name for this role?

Reply:
maybe `ROLES_ADMIN` would be a (slight) improvement?

Reply:
I see!  Think it's fine as-is now that I know, not sure `roles_admin` is any more obvious without the explanation.